README.md

```diff
@@ -7,6 +7,8 @@ In addition to a modern, StimulusReflex-powered user experience, we use [Vite](h
 
 ## Usage
 To get started, create a new Ruby on Rails application, passing in this template with the `-m` option:
+
+_ℹ️ You must have the `pry` gem installed, otherwise line 3 of template.rb will cause an error. Run `gem install pry` to install._
```

Comment:
```suggestion
_ℹ️ You must have the `pry` gem installed, otherwise the line `require 'pry'` in `template.rb` will raise an error. Run `gem install pry` to install the dependency._
```